tests/Functional/DiffTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Tests\Functional;
+
+use Manala\Command\Diff;
+use Manala\Env\EnvEnum;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class DiffTest extends \PHPUnit_Framework_TestCase
+{
+    const ACME_PROJECT_WD = MANALA_DIR.'/tests/fixtures/Command/DiffTest/AcmeProject';
+    const EXPECTED_OUTPUT_FILE = MANALA_DIR.'/tests/fixtures/Command/DiffTest/expected_acme.patch';
```

Comment:
As soon as we always install the exact same version.

I thought about that, but I judge it was too much for the need and will slow down the test suite.